lib/clevertap_plugin.dart

```diff
@@ -39,26 +40,31 @@ class CleverTapPlugin {
       cleverTapPushAmpPayloadReceivedHandler;
   late CleverTapPushClickedPayloadReceivedHandler
       cleverTapPushClickedPayloadReceivedHandler;
-  late CleverTapPushPermissionResponseReceivedHandler cleverTapPushPermissionResponseReceivedHandler;
+  late CleverTapPushPermissionResponseReceivedHandler
+      cleverTapPushPermissionResponseReceivedHandler;
   static List<CleverTapOnVariablesChangedHandler>
       cleverTapOnVariablesChangedHandlers = [];
-  static List<CleverTapOnValueChangedHandler>
-      cleverTapOnValueChangedHandlers = [];
+  static List<CleverTapOnValueChangedHandler> cleverTapOnValueChangedHandlers =
+      [];
 
-  static const MethodChannel _dartToNativeMethodChannel = const MethodChannel('clevertap_plugin/dart_to_native');
-  static const MethodChannel _nativeToDartMethodChannel = const MethodChannel('clevertap_plugin/native_to_dart');
+  static const MethodChannel _dartToNativeMethodChannel = const MethodChannel(
+      'clevertap_plugin/dart_to_native', const StandardMethodCodec());
```

Comment:
Do we need `StandardMethodCodec()` for dart to web communication?

Reply:
Removed `StandardMethodCodec()`